index.html

```diff
@@ -22,6 +22,11 @@
       <a class="icon-Union">
         <i href="#" class="fa fa-bars"></i>
       </a>
+      <ul class="headerlist">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
```

Comment:
- [ ] As requested by previous reviewer please make sure that you stick to the design as much as possible. Your nav link is totally different so I would suggest you please use `color: #344563;` and remove the underline from `li` tag.
### Your design
![his](https://user-images.githubusercontent.com/81727988/236620726-8bdd851e-7065-47b0-a4b9-ff11a26b23a2.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/236620732-021744c6-38b6-408b-b56d-9118b7434217.png)